app/services/host_inventory_api.rb

```diff
@@ -6,59 +6,34 @@
 # Interact with the Insights Host Inventory. Usually HTTP calls
 # are all that's needed.
 class HostInventoryAPI
-  def initialize(id, hostname, account, url, b64_identity)
+  def initialize(id, account, url, b64_identity)
     @id = id
-    @hostname = hostname
     @url = "#{URI.parse(url)}#{ENV['PATH_PREFIX']}/inventory/v1/hosts"
     @account = account
     @b64_identity = b64_identity || @account.b64_identity
   end
 
-  def host_already_in_inventory(hostname_or_id)
+  def host_already_in_inventory(id)
     response = Platform.connection.get(
-      @url, {
-        hostname_or_id: hostname_or_id
-      },
-      X_RH_IDENTITY: @b64_identity
+      "#{@url}/#{id}", {}, X_RH_IDENTITY: @b64_identity
```

Comment:
This check is much simpler than before - I could remove the argument and use the instance variable but it's handy to be able to reuse the same `HostInventoryAPI` object to check for any inventory ID :smile: